### PR TITLE
Adding miceforest

### PR DESCRIPTION
**Pull request by AnotherSamWilson**
Checklist
- [x ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x ] Source is from official source.
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [x ] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/miceforest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - scikit-learn
+    - numpy
+    - pandas
+    - seaborn >=0.11.0
+    - matplotlib >=3.3.0
```
Comment (dopplershift) [inline recipes/miceforest/meta.yaml:27]: Unless you need a pyqt GUI for this package, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base >=3.3.0
```

**Review by dopplershift — COMMENTED** [review]
Recipe is in great shape, just the question about the matplotlib dep.

File: recipes/miceforest/meta.yaml (hunk 1 above)
Comment (AnotherSamWilson) [reply recipes/miceforest/meta.yaml:27]: matplotlib-base doesn't appear to be a package on pypi - will everything run smoothly if there are 2 different dependencies, one for pip and one for conda? No GUIs are used.

Comment (mariusvniekerk) [reply recipes/miceforest/meta.yaml:27]: matplotlib-base is a slightly slimed down version that doesn't depend on qt.

**Comment by AnotherSamWilson** [comment]
@dopplershift Ready for review / merge

**Review by dopplershift — APPROVED** [review]

**Pull request merged**